compiler/src/yul/mappers/expressions.rs

```diff
@@ -38,6 +38,28 @@ pub fn expr(context: &Context, exp: &Spanned<fe::Expr>) -> Result<yul::Expressio
     }
 }
 
+
+pub fn expr_call(context: &Context, exp: &Spanned<fe::Expr>) -> Result<yul::Expression, CompileError> {
+    if let fe::Expr::Call { args, func } = &exp.node {
+        if let fe::Expr::Attribute { value: _, attr } = &func.node {
+            let arguments = &args.node;
+            if let fe::CallArg::Arg(fe::Expr::Num(num)) = arguments[0].node {
+                // let yul_arg = expr(context, val)?;
+                let func_arg = literal_expression! {(num)};
+                let func_name = identifier! { (attr.node) };
+                return Ok(expression! { [func_name]([func_arg])});
```

Comment:
@g-r-a-n-t  I'm working on a follow up to support arguments to call expressions but I'm wondering how to dynamically build yul expressions for a dynamic number of arguments (as opposed to the hardcoded one for a single argument seen above)

Do we already have an example for something like that in the code base?

Reply:
Ok, figured out how to do that. I bypassed the macros for now. Will change it back if there's a macro way to do that :)